lib/oper1.g

```diff
@@ -360,6 +360,40 @@ BIND_GLOBAL( "InstallOtherMethod",
     end );
 
 
+#############################################################################
+##
+#F  InstallEarlyMethod( <opr>,<method> )
+##
+##  <#GAPDoc Label="InstallEarlyMethod">
+##  <ManSection>
+##  <Func Name="InstallEarlyMethod" Arg="opr,method"/>
+##
+##  <Description>
+##  installs a special "early" function method <A>method</A> for the
+##  operation <A>opr</A>. An early method is special in that it bypasses
+##  method dispatch, and is always the first method to be called when
+##  invoking the operation. This can be used to avoid method selection
+##  overhead for certain special cases.
+##  <P/>
+##  Unlike with regular methods, no checks are performed on the
+##  arguments. Hence any operation can have at most one such early
+##  method for each arity (i.e., one early method taking 1 argument, one
+##  early method taking 2 arguments, etc.).
```

Comment:
Somewhere here we should point out that early methods are dangerous and if they handle something, then the user cannot override this behaviour later one with regular method installations

Reply:
I've now added some text for this.